src/zino/zino.py

```diff
@@ -35,7 +36,12 @@ def main():
         level=logging.INFO if not args.debug else logging.DEBUG,
         format="%(asctime)s - %(levelname)s - %(name)s (%(threadName)s) - %(message)s",
     )
-    state.state = state.ZinoState.load_state_from_file() or state.ZinoState()
+    state.config = read_configuration(args.config_file)
```

Comment:
Here's also a small user-friendliness annoyance: If there is a problem with the config, a full traceback is thrown in the user's face.   Exceptions here should be easy to handle and convert to a more user-friendly feedback.   Throwing a traceback at the user usually indicate some deeper unhandled problem in the code and the traceback should be sent to the developers for debugging...